src/common/util/submit-data.js

```diff
@@ -20,7 +20,7 @@ submitData.jsonp = function jsonp (url, jsonp) {
         // for now theres no other way to execute the callback from ingest without jsonp, or unsafe eval / new Function calls
         // future work needs to be conducted to allow ingest to return a more traditional JSON API-like experience for the entitlement flags
         submitData.xhrGet(url + '&jsonp=' + jsonp)
```

Comment:
RE: existing code, was there a reason behind using `importScript` in this try-catch rather than just XHR regardless of env? All workers have XHR but not all have `importScript` function--can classic worker not just use XHR too?

Reply:
importScripts allows the JSONP response to execute (the flags returned by the rum response)

Reply:
Ah ok, just note that still doesn't work for module scripts though. Is the xhr in catch pointless (atm) then if it fails?

Reply:
Nvm, it's the whole "RUM needed to refresh index" isn't it.

Reply:
yeah, basically it tries to use a method that could utilize the jsonp response, but if it can't it falls back to just making the call to set the indexing and capture a "blind" PVE in that it can't get use the setToken call in a safe way